build.sbt

```diff
@@ -98,6 +98,8 @@ lazy val `surge-docs` = (project in file("modules/surge-docs"))
   .dependsOn(`surge-common`, `surge-engine-command-core`, `surge-engine-command-javadsl`, `surge-engine-command-scaladsl`, `surge-metrics`)
   .enablePlugins(ParadoxPlugin, ParadoxSitePlugin, GhpagesPlugin)
   .settings(
+    javacOptions ++= Seq("-source", "16"),
```

Comment:
@satyender071 One more change needed. Can you change this line to javacOptions ++= Seq("-source", "15", "--enable-preview")

Reply:
done